integrations/class-wp-service-worker-scripts-integration.php

```diff
@@ -72,11 +72,26 @@ public function register( WP_Service_Worker_Scripts $scripts ) {
 				$url = wp_scripts()->base_url . $url;
 			}
 
-			$revision = false === $dependency->ver ? get_bloginfo( 'version' ) : $dependency->ver;
-
 			/** This filter is documented in wp-includes/class.wp-scripts.php */
 			$url = apply_filters( 'script_loader_src', $url, $handle );
 
+			$revision = null;
+			$version  = '';
+
+			if ( null === $dependency->ver ) {
+				$revision = wp_scripts()->default_version;
+			} else {
+				$version = $dependency->ver ? $dependency->ver : wp_scripts()->default_version;
+			}
+
+			if ( isset( wp_scripts()->args[ $handle ] ) ) {
+				$version = $version ? $version . '&amp;' . wp_scripts()->args[ $handle ] : wp_scripts()->args[ $handle ];
```

Comment:
I just realized that in the context of the service worker, the `&amp;` here should be `&` since the resulting URL will not be output to HTML for the browser to decode `&amp;` back to `&`:

```suggestion
				$version = $version ? $version . '&' . wp_scripts()->args[ $handle ] : wp_scripts()->args[ $handle ];
```